packages/sku/config/webpack/webpack.config.js

```diff
@@ -43,10 +40,8 @@ const {
   externalizeNodeModules,
 } = config;
 
-// port is only required for dev builds
 const makeWebpackConfig = ({
   isIntegration = false,
-  port = 0,
```

Comment:
Can you clarify the removal of `port`? Not immediately obvious from the diff

Reply:
`port` isn't specified anywhere `makeWebpackConfig` is called, and the only usage of it was in the now-deleted `stringifyEnvarValues`.

I don't think anyone is depending on `process.env.PORT` inside their app code. A high-level look at [a github search for `process.env.PORT`](https://github.com/search?q=org%3ASEEK-Jobs+NOT+is%3Aarchived+process.env.PORT&type=code&p=1) aligns with that assumption.